src/components/_button.scss

```diff
@@ -49,13 +53,20 @@
     @include sui-button--full;
   }
 
-  @if $color != null {
-    @include sui-button--custom($color);
+  @if $bgcolor != null and $c-color != null {
```

Comment:
I don't see the point of making both conditions required. I'd rather put an `or` condition, in order to be able to print a button with only a custom text or background color.